lib/potassium/recipes/background_processor.rb

```diff
@@ -26,6 +28,35 @@ def installed?
     gem_exists?(/sidekiq/)
   end
 
+  private
+
+  def add_docker_compose_redis_config
+    compose = DockerHelpers.new('docker-compose.yml')
+
+    service_definition =
+    <<~YAML
+      image: redis
+      ports:
+        - 6379
+      volumes:
+        - redis_data:/data
+    YAML
+
+    compose.add_service('redis', service_definition)
+    compose.add_volume('redis_data')
+  end
+
+  def set_redis_dot_env
+    append_to_file('.env.development',
+      <<~TEXT
+
```

Comment:
Empty line detected around arguments.<br>Use 2 spaces for indentation in a heredoc.